spec/models/expression_spec.rb

```diff
@@ -158,34 +158,34 @@
   end
 
   describe '#name_attr' do
-    let(:expected_name) do
-      'branch[conditionals_attributes][1][expressions_attributes][0][answer]'
-    end
+    let(:attributes) { %w[answer component operator field] }
 
-    it 'constucts the correct name attribute' do
-      expect(
-        expression.name_attr(
-          conditional_index: 1,
-          expression_index: 0,
-          attribute: 'answer'
-        )
-      ).to eq(expected_name)
+    it 'constructs the correct name attribute' do
+      attributes.map do |attribute|
```

Comment:
`map` returns an array object, but it's never being accessed. the `expect` sits inside the block in which case an `each` would be find if you want to check each attribute expectation

Reply:
Done!